lib/views/drawer/login_view.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:game_lib_app/blocs_and_cubits/drawer_cubit.dart';
+import 'package:game_lib_app/blocs_and_cubits/signin/signin_cubit.dart';
+import 'package:game_lib_app/blocs_and_cubits/user_cubit.dart';
+import 'package:game_lib_app/widgets/error_dialog.dart';
+import 'package:get/get.dart';
+
+class LoginView extends StatefulWidget {
+  const LoginView({Key? key}) : super(key: key);
+
+  @override
+  State<LoginView> createState() => _LoginViewState();
+}
+
+class _LoginViewState extends State<LoginView> {
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  AutovalidateMode _autovalidateMode = AutovalidateMode.disabled;
+  String? _email, _password;
+
+  void _submit() async {
+    setState(() {
+      _autovalidateMode = AutovalidateMode.always;
+    });
+
+    final form = _formKey.currentState;
+
+    if (form == null || !form.validate()) {
+      return;
+    }
+
+    form.save();
+    await context
+        .read<SigninCubit>()
+        .signin(email: _email!, password: _password!)
+        .then((value) => {
+              context
+                  .read<UserCubit>()
+                  .getUser(FirebaseAuth.instance.currentUser?.uid)
+            });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return WillPopScope(
+        onWillPop: () async {
+          return false;
+        },
+        child: BlocConsumer<SigninCubit, SigninState>(
+          listener: (context, state) {
+            if (state.signinStatus == SigninStatus.error) {
+              errorDialog(context, state.error);
+            }
+            if (state.signinStatus == SigninStatus.success) {
+              context.read<DrawerCubit>().changeToInitialState();
+            }
+          },
+          builder: (context, state) {
+            return Column(
+              children: [
+                Align(
+                  alignment: Alignment.topLeft,
+                  child: IconButton(
+                      icon: const Icon(Icons.arrow_back_ios_new_rounded),
+                      onPressed: () => state.signinStatus !=
+                              SigninStatus.submitting
+                          ? context.read<DrawerCubit>().changeToInitialState()
+                          : null),
+                ),
+                const SizedBox(
+                  height: 150,
+                ),
```

Comment:
If you got SizedBox with only one property, remove comma after value and make it one-liner.